cmd/pulumi-test-language/test_host.go

```diff
@@ -243,6 +246,8 @@ func (h *testHost) SignalCancellation() error {
 }
 
 func (h *testHost) Close() error {
```

Comment:
just curious--why do we have multiple entities closing the test host?

Reply:
When the [plugin context is closed it also closes the host](https://github.com/pulumi/pulumi/blob/tg%2Fjournaling-service/sdk/go/common/resource/plugin/context.go#L162).  And since multiple plugins are using this same host, they all try to close it